.github/ISSUE_TEMPLATE/kubernetes_update.md

```diff
@@ -1,38 +1,51 @@
 ---
-name: Upgrade Kubernetes Version
-about: Issue for updating the Kubernetes version (usually decided in community meetings).
-title: 'Update Kubernetes to {version}'
+name: Upgrade Supported Kubernetes Versions
+about: Issue for updating the supported Kubernetes versions.
+title: 'Update Supported Kubernetes to {version_1} {version_2} {version_3}'
 labels: kind/operations, kind/breaking
 assignees: ''
 
 ---
 
-List of items to do for upgrading to {version}:
+Agones supports and is tested against 3 releases of Kubernetes, targeting the newest version as being the [default version in the GKE Rapid channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions). The vendored version of client-go will be aligned with the middle of the three supported Kubernetes versions ({version_2}). All the example clusters will use the middle of the three supported Kubernetes versions ({version_2}).
 
-- [ ] Update terraform submodules
-    - [ ] GKE
-    - [ ] Azure
-    - [ ] EKS
-- [ ] Update e2e cluster
-    - [ ] Recreate cluster with new scripts: `cd build/terraform/e2e; terraform apply -var project=agones-images`
-- [ ] Update kubectl in dev tooling
+List of items to do for upgrading to {version_1} {version_2} {version_3}
+
+- [ ] Update the cluster version of terraform submodules in `install/terraform/modules`
+    - [ ] Update Kubernetes version of GKE cluster to {version_2}
+    - [ ] Update Kubernetes version of AKS to the newest supported version in {version_1} {version_2} {version_3}
+    - [ ] Update Kubernetes version of EKS to the newest supported version in {version_1} {version_2} {version_3}
+- [ ] Update kubectl in dev tooling to {version_2}
     - [ ] Update kubectl in `build/build-image/Dockerfile`
     - [ ] Update kubectl in `build/e2e-image/Dockerfile`
-- [ ] Update documentation for creating clusters
-    - [ ] Config.toml `supported_k8s` and related (do `dev_` before main)
-- [ ] Update the dev tooling to create clusters
-    - [ ] Minikube
-    - [ ] Kind
-- [ ] Update the k8s image used in the helm [pre-delete-hook](https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/hooks/pre_delete_hook.yaml)
-- [ ] Update client-go
-- [ ] Update CRD API reference
+- [ ] Update the Kubernetes version of the below test clusters to {version_2}
+    - [ ] Minikube (Get the patch version [here](https://github.com/kubernetes/minikube/blob/master/CHANGELOG.md))
+    - [ ] Kind (Confirm {version_2} is supported and get the patch version [here](https://github.com/kubernetes-sigs/kind/releases))
+- [ ] Update the k8s image used in the helm [pre-delete-hook](https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/hooks/pre_delete_hook.yaml) to {version_2}
+- [ ] Update client-go in `go.mod` and `test/terraform/go.mod` to {version_2} and re-run `go mod tidy` and `go mod vendor`
```

Comment:
nit: I always have to remind myself which command to run to update client go (since we shouldn't be editing `go.mod` directly). Can you insert the command here since you've added the `go mod tidy` command?

Reply:
Oh! I manually updated the client-go version last time! I will dig more and add the command here.